pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2085,6 +2085,13 @@ public class ServiceConfiguration implements PulsarConfiguration {
     )
     private double loadBalancerMsgThroughputMultiplierDifferenceShedderThreshold = 4;
 
+    @FieldContext(
+            dynamic = true,
+            category = CATEGORY_LOAD_BALANCER,
+            doc = "For each uniform balanced unload, the maximum number of bundles that can be unloaded."
+    )
+    private int maxUnloadBundleNumPerShedding = 2;
```

Comment:
We may need to write a PIP to explain this change.

Reply:
The purpose of adding this configuration is to control the number of bundles unloaded at one time, and prevent too many bundles unloaded at one time, resulting in cluster instability.
@Demogorgon314

Reply:
This is just to add a configuration, the purpose is to control the maximum number of bundles in one round of equalization adjustment.

Reply:
@Demogorgon314 I changed the default value of this configuration to -1. By default, there is no limit to the maximum number of bundles that can be adjusted each time, so it has no effect on the original logic:
<img width="940" alt="image" src="https://user-images.githubusercontent.com/19296967/179389389-93a1434a-ea77-4817-94bf-bf083e486cf1.png">

Reply:
> The purpose of adding this configuration is to control the number of bundles unloaded at one time, and prevent too many bundles unloaded at one time, resulting in cluster instability.

Ok,  you can add this purpose to the `Motivation ` part. Also, I believe we still need to add the unit test.

Reply:
OK. I will fix

Reply:
fixed. PTAL,thanks! @Demogorgon314